docker/development-easy/docker-compose.yml

```diff
@@ -55,9 +55,11 @@ services:
       EASY_DEV_MODE_NEW: "yes"
       DEVELOPER_TOOLS: "yes"
       XDEBUG_ON: 1
+      XDEBUG_MODE: coverage,debug,profile
```

Comment:
What're we using debug and profile for?

Reply:
Debug and profile are used for phpstorm / vscode, we can remove it from the ci dockers, but since I was testing this on the easy development docker, I wanted to make sure we could attach the debugger and run the profiler.  The environment variable overrides the php.ini setting from what I read.